src/main/java/de/cyface/collector/Application.java

```diff
@@ -58,7 +61,10 @@
      *            about supported arguments.
      */
     public static void main(final String[] args) {
-        System.setProperty("vertx.logger-delegate-factory-class-name", "io.vertx.core.logging.SLF4JLogDelegateFactory");
+        System.setProperty("logback.configurationFile", "/home/muthmann/Projekte/Cyface/data-collector/src/main/resources/logback.xml");
```

Comment:
`home/muthmann/Projekte` doesn't sound right for a service ...

Reply:
true